providers/standard/src/airflow/providers/standard/models/__init__.py

```
@@ -0,0 +1,91 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, Any
+
+import sqlalchemy_jsonfield
+from sqlalchemy import Boolean, Column, ForeignKeyConstraint, Integer, MetaData, String, Text
+from sqlalchemy.dialects import postgresql
+from sqlalchemy.orm import registry, relationship
+
+from airflow.models.base import _get_schema, naming_convention
+from airflow.settings import json
+from airflow.utils import timezone
+from airflow.utils.sqlalchemy import UtcDateTime
+
+metadata = MetaData(schema=_get_schema(), naming_convention=naming_convention)
+mapper_registry = registry(metadata=metadata)
+
+if TYPE_CHECKING:
+    Base = Any  # type: ignore[misc]
+else:
+    Base = mapper_registry.generate_base()
+
+
+class HITLInputRequestModel(Base):
+    """Human-in-the-loop input request."""
+
+    __tablename__ = "hitl_input_request"
+    id = Column(Integer, primary_key=True, autoincrement=True)
+    options = Column(sqlalchemy_jsonfield.JSONField(json=json), nullable=False, default=[])
```

I would propose not providing a default, [] would not be usable later. Rather raise an exception if no options provided.

---

sure, will do. Thanks!